pyproject.toml

```diff
@@ -9,6 +9,10 @@ keywords = [
 	"r",
 	"dataset",
 ]
+authors = [
+  {name = "Carlos Ramos Carreño", email = "vnmabus@gmail.com"},
+  {name = "Tuomas Rossi", email = "tuomas.rossi@csc.fi"},
```

Comment:
I agree that CONTRIBUTORS.md could indeed be better place to include this information (related also to the copyright notice comment above). The allcontributors seems a nice idea and using that instead of a manually managed CONTRIBUTORS file is fine with me.

(I find the formatting of [the file](https://github.com/vnmabus/dcor/blob/develop/CONTRIBUTORS.md?plain=1) created by allcontributors somewhat cumbersome due to html controlling visual look instead of containing just the actual content in a more human-readable form, so a cleaner file could be nicer in that sense. But both ways have their pros and cons, and the choice affects also your maintenance work, so it's up to you to decide.)

Reply:
> (I find the formatting of [the file](https://github.com/vnmabus/dcor/blob/develop/CONTRIBUTORS.md?plain=1) created by allcontributors somewhat cumbersome due to html controlling visual look instead of containing just the actual content in a more human-readable form, so a cleaner file could be nicer in that sense. But both ways have their pros and cons, and the choice affects also your maintenance work, so it's up to you to decide.)

There is a cleaner/readable file: the `.all-contributorsrc` is a JSON containing all the information, while `CONTRIBUTORS.md` is automatically generated from it. It is also possible to update the JSON automatically using a bot and mentioning a user in a Github comment. My only concern with allcontributors is the lack of activity on the Github in the last months, but it may be something temporary.

Reply:
Thanks for clarification. Would there be a way to mention the copyright holder (our company in the case of these contributions) within allcontributors? If not (and if the future of that project is uncertain also), I'd suggest to have a plain AUTHORS file instead ([the example here](https://opensource.google/documentation/reference/releasing/authors) seems nice with a preamble that the revision history has the detailed information). I think this would be more future-proof as it's much simpler than json + html generated by a bot. What do you think?

Reply:
I can mention both the company and you if you want. I used to have a plain AUTHORS file in other project, but I think that allcontributors is better, as it can be shown in the documentation too, and is a positive feedback for contributors.

Reply:
I added the support for allcontributors. You can see the rendered docs [here](https://rdata.readthedocs.io/en/latest/contributors.html). I added both you and your employer. Note that the latter links to its official website, and its code contributions emoji links to yours. Please tell me if that is enough, and if you want to be listed also in pyproject.toml or not.